packages/katello/katello-repos/katello-repos.spec

```
@@ -68,6 +68,9 @@ rm -rf %{buildroot}
 %config %{repo_dir}/*.repo
 
 %changelog
+* Thu Jul 2 2021 James Jeffers <jjeffers@redhat.com> - 4.2-0.1.nightly
```

You missed the release bump (line 8). Then this needs to match
```suggestion
* Thu Jul 2 2021 James Jeffers <jjeffers@redhat.com> - 4.2-0.2.nightly
```